packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -360,7 +360,10 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 			signalHandler: (message: ISignalMessage) => this._inboundSignal.push(message),
 			reconnectionDelayHandler: (delayMs: number, error: unknown) =>
 				this.emitDelayInfo(this.deltaStreamDelayId, delayMs, error),
-			closeHandler: (error: any) => this.close(error),
+			closeHandler: (error: any) => {
+				this.close(error);
+				this.close(error, true);
```

Comment:
This looks fishy. Why do we need two calls to close()? Is it a problem of not exposing better APIs (even if they are calling into same core logic)?